src/state/lists/updater.ts

```diff
@@ -27,8 +34,8 @@ export default function Updater(): null {
     )
   }, [fetchList, isWindowVisible, lists])
 
-  // fetch all lists every 10 minutes, but only after we initialize library
-  useInterval(fetchAllListsCallback, library ? 1000 * 60 * 10 : null)
+  // fetch all lists every 10 minutes, but only after we initialize library and page has currency input
+  useInterval(fetchAllListsCallback, library ? 1000 * 60 * 10 : null, true, includeListUpdater)
```

Comment:
can we simplify this as
```js
useInterval(fetchAllListsCallback, library && includeListUpdater ? 1000 * 60 * 10 : null, includeListUpdater)
```

Reply:
No, we need to run interval on the background even includeListUpdater is false to not run every time user visits swap pages.

Reply:
Could we do this?

```
useInterval(() => includeListUpdater && fetchAllListsCallback(), ...)
```

Reply:
```
const fetchAllListsCallback = useCallback(() => {
  if (!isWindowVisible || includeListUpdater) return
  ....
})
```

Reply:
Done